app/views/idv/doc_auth/_ssn_init.html.erb

```diff
@@ -46,3 +46,4 @@
   </div>
 <% end %>
 <%= render 'idv/doc_auth/start_over_or_cancel', step: 'ssn' %>
+<%= javascript_packs_tag_once('ssn-field') %>
```

Comment:
the ssn field markup is basically identical (same form field name, same translation) to the one in `ssn_update`, if we're going to have a separate JS include for this file would it make sense to extract a partial for that field and put the JS pack tags in there?

Reply:
> the ssn field markup is basically identical (same form field name, same translation) to the one in `ssn_update`, if we're going to have a separate JS include for this file would it make sense to extract a partial for that field and put the JS pack tags in there?

Yeah, I think we might be able to consolidate to one partial between creating and updating SSN. Even then, it could still make sense to extract a partial just for the input field as well, so that the markup and JavaScript are packaged as a self-contained unit.

Reply:
I extracted the SSN field in b9ca762. I started consolidating the SSN views as well, but it's a bit more work than I'd like to take on here, so probably worth a follow-up.